src/tail/mod.rs

```diff
@@ -33,3 +49,14 @@ impl Tail {
         })
     }
 }
+
+async fn handle_sigint(txs: Vec<oneshot::Sender<()>>) -> Result<(), failure::Error> {
+    tokio::signal::ctrl_c().await?;
+    for tx in txs {
+        if let Err(e) = tx.send(()) {
+            eprintln!("failed to transmit to channel {:?}", e);
```

Comment:
need a better error message here or panic since it's an unrecoverable bug

Reply:
i'm going to link this in my larger error handling issue, because i need to look at the various types of errors i could receive here.

Reply:
https://github.com/cloudflare/wrangler/issues/1177